code/game/machinery/computer/HolodeckControl.dm

```diff
@@ -399,6 +399,14 @@ var/global/list/holodeck_programs = list(
 	desc = "Here's your chance, do your dance at the Space Jam."
 	w_class = 4 //Stops people from hiding it in their bags/pockets
 
+	//Can be picked up by aliens
+/obj/item/toy/beach_ball/holoball/attack_paw(user as mob)
+	if(isxeno(user))
```

Comment:
Prefer early returns instead of nested logic.

Reply:
That means
```if(!isxeno(user))
	return FALSE
attack_alien(user)
```
The point to this is to diminish unnecessary indentation as procs grow complex.